sasdata/quantities/units.py

```diff
@@ -0,0 +1,3415 @@
+        "ES": exasiemens,
+        "PS": petasiemens,
+        "TS": terasiemens,
+        "GS": gigasiemens,
+        "MS": megasiemens,
+        "kS": kilosiemens,
+        "mS": millisiemens,
+        "uS": microsiemens,
+        "µS": microsiemens,
+        "nS": nanosiemens,
+        "pS": picosiemens,
+        "fS": femtosiemens,
+        "aS": attosiemens,
+        "Wb": webers,
+        "EWb": exawebers,
+        "PWb": petawebers,
+        "TWb": terawebers,
+        "GWb": gigawebers,
+        "MWb": megawebers,
+        "kWb": kilowebers,
+        "mWb": milliwebers,
+        "uWb": microwebers,
+        "µWb": microwebers,
+        "nWb": nanowebers,
+        "pWb": picowebers,
+        "fWb": femtowebers,
+        "aWb": attowebers,
+        "T": tesla,
+        "ET": exatesla,
+        "PT": petatesla,
+        "TT": teratesla,
+        "GT": gigatesla,
+        "MT": megatesla,
+        "kT": kilotesla,
+        "mT": millitesla,
+        "uT": microtesla,
+        "µT": microtesla,
+        "nT": nanotesla,
+        "pT": picotesla,
+        "fT": femtotesla,
+        "aT": attotesla,
+        "H": henry,
+        "EH": exahenry,
+        "PH": petahenry,
+        "TH": terahenry,
+        "GH": gigahenry,
+        "MH": megahenry,
+        "kH": kilohenry,
+        "mH": millihenry,
+        "uH": microhenry,
+        "µH": microhenry,
+        "nH": nanohenry,
+        "pH": picohenry,
+        "fH": femtohenry,
+        "aH": attohenry,
+        "Ang": angstroms,
+        "Å": angstroms,
+        "min": minutes,
+        "h": hours,
+        "d": days,
+        "y": years,
+        "deg": degrees,
+        "rad": radians,
+        "sr": stradians,
+        "l": litres,
+        "eV": electronvolts,
+        "EeV": exaelectronvolts,
+        "PeV": petaelectronvolts,
+        "TeV": teraelectronvolts,
+        "GeV": gigaelectronvolts,
+        "MeV": megaelectronvolts,
+        "keV": kiloelectronvolts,
+        "meV": millielectronvolts,
+        "ueV": microelectronvolts,
+        "µeV": microelectronvolts,
+        "neV": nanoelectronvolts,
+        "peV": picoelectronvolts,
+        "feV": femtoelectronvolts,
+        "aeV": attoelectronvolts,
+        "au": atomic_mass_units,
+        "mol": moles,
+        "mmol": millimoles,
+        "umol": micromoles,
+        "µmol": micromoles,
+        "nmol": nanomoles,
+        "pmol": picomoles,
+        "fmol": femtomoles,
+        "amol": attomoles,
+        "kgForce": kg_force,
+        "miles": miles,
+        "yrd": yards,
+        "ft": feet,
+        "in": inches,
+        "lb": pounds,
+        "lbf": pounds_force,
+        "oz": ounces,
+        "psi": pounds_force_per_square_inch,
+        "percent": percent,
+        "%": percent,
+        "yr": years,
+        "year": years,
+        "day": days,
+        "hr": hours,
+        "hour": hours,
+        "a.u.": atomic_mass_units,
+        "amu": atomic_mass_units,
+        "degr": degrees,
+}
+
+
+#
+# Units by type 

```

Comment:
While writing the unit tests for my parser, I had a thought. I think the unit groups should perhaps go into a separate module. To import a unit, we would do

```python
from sasdata.quantities.units import meters. meters_per_second ...'
```

Which makes sense. But to import a unit group, the import statement would look like this.

```python
from sasdata.quantities.units import speed, acceleration ...'
```

Which might be a little confusing because we're importing a unit *group* from the units module. Its a fairly minor change but having a separate unit group module might make this clearer.

Reply:
Yeah, sounds like a good plan